feat(howto): detail runtime log display (#66)
anbox-cloud/discourse-docs: howto/container/logs.md

> ## Runtime logs

Reviewer: ```suggestion
You can view the Anbox and the Android system logs while a container is running, or check the collected logs if a container fails.
[note type="information" status="Note"]AMS does not support runtime log collection. Logs are currently only being collected from a container that failed to start or had an error at runtime.[/note]

## View runtime logs
```

Reviewer: This moves the note from the bottom up. Do I understand it correctly that the note is still true with the addition in this PR, but that runtime logs can only be **viewed** and are not **collected**?

Author: We can view runtime logs (while the container is running) and collected logs (when the container is stopped). 

Makes sense to drop the note.

> At runtime of a container the `amc` commands provides access to both the Anbox and Android system logs. With the following command the logs of both can be viewed:
> 
>     amc logs -t <log type>
> 
> As `<log type>` you can either specify `anbox` for the Anbox system log or `android` for the Android one. In case that you don't specify the log type by default the Anbox system log will be displayed.

Reviewer: ```suggestion
While a container is running, you can use the `amc` command to access both the Anbox and the Android system logs. The following command shows the system log:

    amc logs -t <log type>

As `<log type>`, specify either `anbox` for the Anbox system log or `android` for the Android one. If you don't specify the log type, the Anbox system log is displayed by default.
```

> In case that you want to follow the log and get automatically updated for new entries you can add the `-f` argument:
> 
>     amc logs -t anbox -f
> 
> This will view the logs and update the output whenever new entries are added.

Reviewer: ```suggestion
To follow the log and get automatic updates for new entries, add the `-f` argument:

    amc logs -t anbox -f

This will show the logs and update the output whenever new entries are added.
```

> ## Stored logs

Reviewer: ```suggestion
## View stored logs
```

If a container fails to start or a runtime error occurs, AMS collects relevant log files from the container and makes them available for inspection.

> Available logs can be listed with the `amc show <container_id>` command:

Reviewer: Remove the note at the bottom.

Author: Done

You can view the Anbox and the Android system logs while a container is running, or check the collected logs if a container fails.

> [note type="information" status="Note"]AMS does not support runtime log collection. Logs are currently only being collected from a container that failed to start or had an error at runtime.[/note]

Reviewer 2: Would it be worth also noting that logs can be fetched manually via logcat ?
